pulp_ansible/tests/functional/utils.py

```diff
@@ -63,7 +63,7 @@ def gen_ansible_remote(url=ANSIBLE_FIXTURE_URL, **kwargs):
     :param url: The URL of an external content source.
     """
     if "rate_limit" not in kwargs:
-        kwargs["rate_limit"] = 10
+        kwargs["rate_limit"] = 5
```

Comment:
Is this needed? I thought the rate_limit of 10 was enough for galaxy.ansible.com and c.rh.c syncs?

Reply:
this PR was constantly failing before this change